src/Discovery/DiscoveryListView.tsx

```diff
@@ -47,15 +47,28 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
       columns={props.columns}
       rowKey={props.config.minimalFieldMapping.uid}
       rowSelection={(
-        props.config.features.exportToWorkspace
-              && props.config.features.exportToWorkspace.enabled
+        (props.config.features.exportToWorkspace
+          && props.config.features.exportToWorkspace.enabled) || (props.config.features.exportToWorkspace?.enableFillRequestForm
+            && props.config.features.exportToWorkspace.enableFillRequestForm === true)
       ) && {
         selectedRowKeys: props.selectedResources.map(
           (r) => r[props.config.minimalFieldMapping.uid],
         ),
         renderCell: (_checked, _record, _index, node) => (
           <Tooltip
-            title={`Click to select item for ${(props.config.features.exportToWorkspace.enableDownloadManifest || props.config.features.exportToWorkspace.enableDownloadZip) ? 'download or ' : ''}open in workspace`}
+            title={`Click to select item for ${
+              [
+                props.config.features.exportToWorkspace.enableFillRequestForm
+                  ? props.config.features.exportToWorkspace.fillRequestFormDisplayText.toLowerCase()
```

Comment:
```suggestion
                  ? props.config.features.exportToWorkspace.fillRequestFormDisplayText?.toLowerCase()
```
`fillRequestFormDisplayText` can possibily be `undefined`